src/Symfony/Component/Scheduler/Messenger/ServiceCallMessage.php

```diff
@@ -42,6 +42,17 @@ public function getArguments(): array
 
     public function __toString(): string
     {
-        return "@$this->serviceId".('__invoke' !== $this->method ? "::$this->method" : '');
+        return "@$this->serviceId".('__invoke' !== $this->method ? "::$this->method" : '').$this->getSerializedArguments();
```

Comment:
The value returned by `__toString()` should probably be human-readable, which is not the case anymore.

Reply:
I was more thinking about what you mentioned in your "EDIT" paragraph.
I agree with you that using __toString() for uniqueness is really bad.

Reply:
:point_up: Did just that, now it's like

```
 ----------------- -------------------------------------------------------------------------------------------------------------------------------------------- --------------------------------- 
  Trigger           Provider                                                                                                                                     Next Run                         
 ----------------- -------------------------------------------------------------------------------------------------------------------------------------------- --------------------------------- 
  every 2 seconds   Symfony\Component\Scheduler\Messenger\ServiceCallMessage(@App\MessageHandler\TriggeringMessageHandler::handleTriggeringMessage)              Wed, 06 Dec 2023 12:41:06 +0200  
  every 2 seconds   Symfony\Component\Scheduler\Messenger\ServiceCallMessage(@App\MessageHandler\TriggeringMessageHandler::handleTriggeringMessage(#cc40c189))   Wed, 06 Dec 2023 12:41:06 +0200  
  every 2 seconds   Symfony\Component\Scheduler\Messenger\ServiceCallMessage(@App\MessageHandler\TriggeringMessageHandler::handleTriggeringMessage(#266e01fa))   Wed, 06 Dec 2023 12:41:06 +0200  
```

Can we go with this for 6.4 to fix the issue (which is quite big one in my opinion)?